src/index.js

```diff
@@ -1,2 +1,10 @@
-// add soon
-import './style.css';
\ No newline at end of file
+import './style.css';
+import getscore from '../modules/getscore.js';
+import renderSubmit, { render } from '../modules/render&submit.js';
+// import setgame from '../modules/setgame.js';
+
+// setgame();
```

Comment:
- [ ] Please ensure that you remove the commented code from your codebase to maintain a neat and clean appearance. Remember, you can always refer back to the Git history ✨💪